src/compas_fab/ghpython/components/install.py

```diff
@@ -0,0 +1,38 @@
+import os
+import glob
+from shutil import copyfile
+import compas
+
+from compas_fab.ghpython.components import assembly_name
+from compas_fab.ghpython.components import filename
+
+# TODO: function below must move into compas_ghpython
+
+
+def _get_grasshopper_library_path():
```

Comment:
It might makes sense to add this function (one day)  to the `__init__.py` of `compas_ghpython`.

Reply:
yes indeed ;)